packages/aws-cdk-lib/custom-resources/lib/provider-framework/runtime/outbound.ts

```diff
@@ -9,7 +9,7 @@ import { SFN, StartExecutionInput, StartExecutionOutput } from '@aws-sdk/client-
 const FRAMEWORK_HANDLER_TIMEOUT = 900000; // 15 minutes
 
 // In order to honor the overall maximum timeout set for the target process,
-// the default 2 minutes from AWS SDK has to be overriden:
+// the default 2 minutes from AWS SDK has to be overridden:
```

Comment:
This is a custom resource, presumably this is causing the hash for integ tests to change so the build is failing. Going to take this typo out of this change for now to merge this PR in as a lot of the integ tests are changing
```suggestion
// the default 2 minutes from AWS SDK has to be overriden:
```